### PR TITLE
S127.41 Change exchange to 5 decimals

### PR DESCRIPTION
**Pull request by ronoudgenoeg**

### CODE REVIEW

**Review by mtricht — APPROVED** [review]

**Comment by ronoudgenoeg** [comment]
test this please

**Comment by ronoudgenoeg** [comment]
test this please

**Comment by ronoudgenoeg** [comment]
test this please

File: Config/Schema/schema.php [hunk 1]
```diff
@@ -81,7 +81,7 @@ public function after($event = array()) {
 	public $cper_exchange_rates = array(
```
Comment (expand-dev) [inline Config/Schema/schema.php:81]: MINOR Rename this field "$cper_exchange_rates" to match the regular expression ^[a-z][a-zA-Z0-9]*$. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS116)

File: Config/Schema/schema.php [hunk 2]
```diff
@@ -81,7 +81,7 @@ public function after($event = array()) {
...
 		'id' => array('type' => 'integer', 'null' => false, 'default' => null, 'unsigned' => true, 'key' => 'primary'),
 		'currency' => array('type' => 'string', 'null' => false, 'default' => null, 'length' => 10, 'key' => 'index', 'collate' => 'latin1_swedish_ci', 'charset' => 'latin1'),
```
Comment (expand-dev) [inline Config/Schema/schema.php:83]: MAJOR Split this 169 characters long line (which is greater than 120 authorized). [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS103)

**Comment by expand-dev** [comment]
SonarQube analysis reported 25 issues
* ![MAJOR][MAJOR] 17 major
* ![MINOR][MINOR] 8 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [schema.php#L12](https://github.com/ExpandOnline/CakePHPExchangeRates/blob/2baf8ef23fe7eacd887f24e2fa73f8f50c8062d3/Config/Schema/schema.php#L12): Split this 130 characters long line (which is greater than 120 authorized). [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS103)
1. ![MAJOR][MAJOR] [schema.php#L14](https://github.com/ExpandOnline/CakePHPExchangeRates/blob/2baf8ef23fe7eacd887f24e2fa73f8f50c8062d3/Config/Schema/schema.php#L14): Split this 145 characters long line (which is greater than 120 authorized). [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS103)
1. ![MAJOR][MAJOR] [schema.php#L16](https://github.com/ExpandOnline/CakePHPExchangeRates/blob/2baf8ef23fe7eacd887f24e2fa73f8f50c8062d3/Config/Schema/schema.php#L16): Split this 145 characters long line (which is greater than 120 authorized). [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS103)
1. ![MAJOR][MAJOR] [schema.php#L17](https://github.com/ExpandOnline/CakePHPExchangeRates/blob/2baf8ef23fe7eacd887f24e2fa73f8f50c8062d3/Config/Schema/schema.php#L17): Split this 128 characters long line (which is greater than 120 authorized). [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS103)
1. ![MAJOR][MAJOR] [schema.php#L29](https://github.com/ExpandOnline/CakePHPExchangeRates/blob/2baf8ef23fe7eacd887f24e2fa73f8f50c8062d3/Config/Schema/schema.php#L29): Split this 130 characters long line (which is greater than 120 authorized). [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS103)
1. ![MAJOR][MAJOR] [schema.php#L31](https://github.com/ExpandOnline/CakePHPExchangeRates/blob/2baf8ef23fe7eacd887f24e2fa73f8f50c8062d3/Config/Schema/schema.php#L31): Split this 145 characters long line (which is greater than 120 authorized). [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS103)
1. ![MAJOR][MAJOR] [schema.php#L33](https://github.com/ExpandOnline/CakePHPExchangeRates/blob/2baf8ef23fe7eacd887f24e2fa73f8f50c8062d3/Config/Schema/schema.php#L33): Split this 145 characters long line (which is greater than 120 authorized). [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS103)
1. ![MAJOR][MAJOR] [schema.php#L34](https://github.com/ExpandOnline/CakePHPExchangeRates/blob/2baf8ef23fe7eacd887f24e2fa73f8f50c8062d3/Config/Schema/schema.php#L34): Split this 128 characters long line (which is greater than 120 authorized). [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS103)
1. ![MAJOR][MAJOR] [schema.php#L46](https://github.com/ExpandOnline/CakePHPExchangeRates/blob/2baf8ef23fe7eacd887f24e2fa73f8f50c8062d3/Config/Schema/schema.php#L46): Split this 130 characters long line (which is greater than 120 authorized). [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS103)
1. ![MAJOR][MAJOR] [schema.php#L47](https://github.com/ExpandOnline/CakePHPExchangeRates/blob/2baf8ef23fe7eacd887f24e2fa73f8f50c8062d3/Config/Schema/schema.php#L47): Split this 132 characters long line (which is greater than 120 authorized). [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS103)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**